docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FieldAwareFactorizationMachine.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class FFMBinaryClassification
+    {
+        public static void Example()
+        {
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline.
```

Comment:
>// Creating the ML.Net IHostEnvironment object, needed for the pipeline. [](start = 12, length = 72)

```
           // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
            // as a catalog of available operations and as the source of randomness.
```
I think this is our default comment.